examples/stable_diffusion_v2/controlnet.md

```diff
@@ -73,49 +73,65 @@ Put them in an arbitrary directory on your machine. For example, `path/to/test_i
 
    Attention: As the DeeplabV3Plus is trained on VOC dataset, currently it only supports prompts related to the objects: 'background', 'aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train','tvmonitor'. More is coming soon.
 
-### Image Generation
+### Parameter Setting
+
+Go to the directory `stable_diffusion_v2/inference` first.
+
 1. Canny edge maps:
 
-   Run the command below to generate images
+   Open the file `stable_diffusion_v2/inference/config/controlnet.yaml`. Set arguments as below:
```

Comment:
Please use two separate yaml files for canny control and segmentation control

Reply:
fixed